Tools-Override/tests.targets

```diff
@@ -211,8 +211,54 @@
       <TestCommandLines Include="cd native"/>
     </ItemGroup>
 
+    <!-- For UAP F5 scenario, generate the test commands -->
+    <Error Condition="'$(BuildingUAPVertical)' == 'true' and '$(UAPRunnerFolder)'==''" Text="Please specify UAPRunnerFolder" />
+    <Error Condition="'$(BuildingUAPVertical)' == 'true' and '$(UAPLauncherFolder)'==''" Text="Please specify UAPLauncherFolder" />
+
+    <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true'">
+      <_Runner_Path>$(UAPRunnerFolder)\</_Runner_Path>
+      <_Launcher_Path>$(UAPLauncherFolder)\WindowsStoreAppLauncher.exe</_Launcher_Path>
+      <_Runtime_Path>$(RuntimePath)</_Runtime_Path>
+      <_Results_Path>$([System.Environment]::GetFolderPath(SpecialFolder.MyDocuments))\TestResults\</_Results_Path>
+    </PropertyGroup>
+  
+    <!-- The paths are different on the helix machine than locally -->
+    <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true' and '$(ArchiveTests)' == 'true'">
+      <_Runner_Path>%RUNTIME_PATH%\Runner\</_Runner_Path>
```

Comment:
Don't we have a good way to make these the same for either ArchiveTests or not? Ideally we would still use the RUNTIME_PATH variable in both so we are consistent.